custom/enikshay/tasks.py

```diff
@@ -397,6 +397,14 @@ def _get_fulfilled_vouchers(self):
         ]
         return sorted(relevant_vouchers, key=self._get_fulfilled_voucher_date)
 
+    def _get_fulfilled_available_vouchers(self):
+        relevant_vouchers = [
+            voucher for voucher in self._get_all_vouchers()
+            if (voucher.get_case_property('voucher_type') == 'prescription'
+                and voucher.get_case_property('state') in ['fulfilled', 'available'])
```

Comment:
Yeah, that also sounds reasonable.  `state` feels overloaded, anyways.  It's easy enough to handle in HQ, so I think whatever makes the most sense from an app-building perspective.  The payment confirmation status is set [here](https://github.com/dimagi/commcare-hq/blob/fd8e4b65e86023d37c2482894b45f01be300efca/custom/enikshay/integrations/bets/views.py#L71-L71).
I'm not planning on making any changes right now, but if you or @ctsims would like, I'd be happy to set `payment_status` instead of `state`.